UI/Entities/Game.cs

```diff
@@ -81,13 +81,15 @@ public void RemovePlayer(string connectionId)
         {
           Host.HasWon = true;
           Host.HasTurn = false;
+          Guest.HasTurn = false;
```

Comment:
Why is this needed? This change shouldn't be required.

Reply:
Apologies @Rawan-Hamza, I'm blind! I lost my way and thought this was somewhere else.

Your change above is good, just check if `Guest` is `null` before setting to `false` like so:

```csharp
if (Guest != null) Guest.HasTurn = false;
```

Reply:
No worries!, will action these changes. Thank you